counterpoll/main.py

```diff
@@ -234,3 +235,48 @@ def show():
 
     click.echo(tabulate(data, headers=header, tablefmt="simple", missingval=""))
 
+def _update_config_db(state, filename):
+    """ Update counter configuration in config_db file """
+    with open(filename) as config_db_file:
+        config_db = json.load(config_db_file)
+
+    counters = [
+        "QUEUE_WATERMARK",
+        "BUFFER_POOL_WATERMARK",
+        "PFCWD",
+        "QUEUE",
+        "PG_WATERMARK",
+        "PORT_BUFFER_DROP",
+        "RIF",
+        "PORT",
+    ]
+
+    write_config_db = False
+    if "FLEX_COUNTER_TABLE" in config_db:
+        for counter in counters:
+            if counter in config_db["FLEX_COUNTER_TABLE"] and \
+                "FLEX_COUNTER_STATUS" in config_db["FLEX_COUNTER_TABLE"][counter]:
+                config_db["FLEX_COUNTER_TABLE"][counter]["FLEX_COUNTER_STATUS"] = state
+                write_config_db = True
+
+    if write_config_db:
+        with open(filename, 'w') as config_db_file:
+            json.dump(config_db, config_db_file, indent=4)
+
+# Working on Config DB
+@cli.group()
+def config_db():
```

Comment:
This new command is basically "counterpoll disable && config save -y" ?
I wonder, why not disable counterpolling in running redis db and do config save in the fast-reboot script?

Reply:
This is possible as well. However I did not want to lose visibility of counters while in the reboot process had it failed in a subsequent step and so, opted for it to be done at a later step.